cli/ballerina-cli/src/test/java/io/ballerina/cli/cmd/PackCommandTest.java

```diff
@@ -270,4 +270,32 @@ public void testCustomTargetDirWithRelativePath() throws IOException {
         Assert.assertTrue(customTargetDir.resolve("bala").resolve("foo-winery-any-0.1.0.bala").toFile().exists());
     }
 
+    @Test(description = "Pack an empty package")
+    public void testPackEmptyPackage() throws IOException {
+        Path projectPath = this.testResources.resolve("emptyPackage");
+        System.setProperty("user.dir", projectPath.toString());
+
+        PackCommand packCommand = new PackCommand(projectPath, printStream, printStream, false, true);
+        new CommandLine(packCommand).parse();
+        packCommand.execute();
+
+        String buildLog = readOutput(true);
+        Assert.assertEquals(buildLog.replaceAll("\r", ""),
```

Comment:
`bal pack` on empty package should fail if compiler plugin is not present.

Reply:
When i ran the test individually it did fail since the output was incorrect, but for some reason when I run the tests together, or say if the github tests run, it doesnt fail. @azinneera any clue why? 

I changed the output to the correct one. It shouldn't fail now since i checked it individually but we may be overlooking somethings